src/protocol_splitter.cpp

```diff
@@ -134,20 +134,15 @@ int DevSerial::open_uart()
 		printf("[ protocol__splitter ]\tSerial link: device: %s; baudrate: %d; flow_control: %s\n",
 		       _uart_name, _baudrate, _sw_flow_control ? "SW enabled" : (_hw_flow_control ? "HW enabled" : "No"));
 
-		char aux[64];
-		bool flush = false;
-
-		while (0 < ::read(_uart_fd, (void *)&aux, 64)) {
-			flush = true;
-			usleep(1000);
+		if (tcflush(_uart_fd,TCIOFLUSH) < 0) {
+			int errno_bkp = errno;
+			printf("\033[0;31m[ protocol__splitter ]\tSerial link: Error flushing buffers %s: (%d)\033[0m\n", _uart_name,
```

Comment:
Is there spaces&tabs? Maybe fix those first, then merge

Reply:
Yes, seems that there are both tabs and spaces used everywhere in the file, also in the original upstream version. Looks like tabs are used for normal group intend and then spaces are used for intending some continuing line to align with content above. Should we convert all tabs to spaces then?